Install-Kite.md

```diff
@@ -10,4 +10,22 @@ curl http://central.maven.org/maven2/org/kitesdk/kite-tools/{{site.version}}/kit
 chmod +x {{site.dataset-command}}
 ```
 
-The first command downloads the Kite tools JAR to a local file named _{{site.dataset-command}}_. The second changes the access rights on the JAR so that the classes in the JAR are executable.
+The first command downloads the Kite tools JAR to a local file named `{{site.dataset-command}}`. The second changes the access rights on the JAR so that the classes in the JAR are executable.
+
+Some versions of CDH come with `{{site.dataset-command}}` installed. However, it might not be the latest version. To check the version, from a terminal window, enter the following command.
```

Comment:
Yeah, okay. The reason I hesitate to do that is that I'm pretty sure everyone will always need to do something in order to use the latest version of the software. I'll add the heading "Verifying the Version In Use," which will hopefully get them to keep reading.